common/src/types/token.rs

```diff
@@ -240,6 +321,29 @@ lazy_static! {
         }
         exchange_tickers
     };
+    /// The set of stablecoin quote assets
+    pub static ref STABLECOIN_QUOTES: HashSet<Token> = {
+        let stablecoin_tickers = [USDC_TICKER, USDT_TICKER];
+        stablecoin_tickers.iter().map(|ticker| Token::from_ticker(ticker)).collect()
+    };
+    /// The mapping between exchanges and the stablecoin quote asset on which they are generally
+    /// the most liquid
+    pub static ref MOST_LIQ_STABLE: HashMap<Exchange, Token> = {
+        let mut most_liquid_stable = HashMap::<Exchange, Token>::new();
+
+        // Most liquid stablecoin quote on Binance is generally USDT
+        most_liquid_stable.insert(Exchange::Binance, Token::from_ticker(USDT_TICKER));
+        // Most liquid stablecoin quote on Coinbase is generally USDC
+        most_liquid_stable.insert(Exchange::Coinbase, Token::from_ticker(USDC_TICKER));
+        // Most liquid "stablecoin" quote on Kraken is generally USD
+        most_liquid_stable.insert(Exchange::Kraken, Token::from_ticker(USD_TICKER));
+        // Most liquid stablecoin quote on Okx is generally USDT
+        most_liquid_stable.insert(Exchange::Okx, Token::from_ticker(USDT_TICKER));
+
+        // We currently omit UniswapV3 here, as it requires more involved routing logic
+
+        most_liquid_stable
+    };
```

Comment:
this should probably just be a method, we can make it static if we like, but we should mimize the scope of these `lazy_statics` as much as possible. I.e. something like:
```rust
default_exchange_stable(exchange: Exchange) -> Token {
    match exchange {
        Exchange::Binance => Token::from_ticker(USDT_TICKER),
        ...
    }
}
```

Also notice the rename, "most liquid" is _currently_ how we set the value, but "default" captures a more general notion